File: SocialAnalizeBackend/SocialAnalizeBackend/SAB.Contracts/User/FollowedUser.cs
using GoldenEye.Backend.Core.Entity;
using GoldenEye.Shared.Core.Objects.General;
using SA.Contracts.Reports.Character;
using SA.Contracts.Reports.InternetActivity;
using SA.Contracts.Reports.Localization;
using SA.Contracts.Reports.PersonalData;

namespace SA.Contracts.User
{
    public class FollowedUser : IEntity
    {
        public int Id { get; set; }
        object IHasId.Id => Id;
        public string FacebookId { get; set; }
        public string InstagramId { get; set; }
        public string TwitterId { get; set; }
        public CharacterData CharacterData { get; set; }
        public PersonalData PersonalData { get; set; }
        public Localization LocalizationData { get; set; }
        public InternetActivityData InternetActivityData { get; set; }
        public User User { get; set; }
    }
}

File: SocialAnalizeF/SocialAnalize/src/app/models/users/followed.user.ts
import { PersonalData } from './personal.data';
import { CharacterData } from './character.data';

export class FollowedUser {
    public id: number;
    public facebookId: string;
    public instagramId: string;
    public twitterId: string;
    public personalData: PersonalData;
    public characterData: CharacterData;
}
